internal/migrations/20210822134659_multiple_networks.go

```diff
@@ -1,66 +0,0 @@
-package migrations
-
-import (
-	gormigrate "github.com/go-gormigrate/gormigrate/v2"
-	"github.com/openshift/assisted-service/internal/common"
-	"github.com/openshift/assisted-service/models"
-	"gorm.io/gorm"
-)
-
-func multipleNetworks() *gormigrate.Migration {
```

Comment:
So theoretically we should keep this migration.
Could there be some installation out there that still hasn't run this migration, but will still want to move the cluster data from the old columns to the new tables when they get around to upgrading?

Specifically I wonder if ACM 2.3 is still supported and if people can upgrade directly from 2.3 to 2.6 or something.
I don't know if this migration pre-dates 2.3, but the idea is that it should work effectively forever.